OrbitCore/CoreApp.cpp

```diff
@@ -3,4 +3,4 @@
 //-----------------------------------
 #include "CoreApp.h"
 
-CoreApp* GCoreApp;
\ No newline at end of file
+std::shared_ptr<CoreApp> GCoreApp;
```

Comment:
This will probably need another refactoring where we get rid of all shared_ptrs... I understand that removing this global requires significant changes to codebase but isn't the right way to fix this problem?

Reply:
Can it be changed to unique_ptr, does it have to be shared? It is not passed anywhere as shared_ptr, right?

Reply:
I changed it now in the following way:
GOrbitApp is the unique_ptr which owns the OrbitApp instance and GCoreApp is a regular pointer to the same instance